integration-cli/docker_cli_restart_test.go

```diff
@@ -36,9 +32,7 @@ func (s *DockerSuite) TestRestartRunningContainer(c *check.C) {
 	c.Assert(waitRun(cleanedContainerID), check.IsNil)
 
 	out, _ = dockerCmd(c, "logs", cleanedContainerID)
-	if out != "foobar\n" {
-		c.Errorf("container should've printed 'foobar'")
-	}
+	c.Assert(out, check.Equals, "foobar\n", check.Commentf("container should've printed 'foobar'"))
```

Comment:
Just `c.Assert(out, check.Equals, "foobar\n")` should suffice. The error message on failure will give the needed information in this cases.